### ISSUE TITLE
lalrpop executable does not have --version or -V option

### ISSUE THREAD

**Issue by Dragonrun1** [nikomatsakis/lalrpop#224]
Be nice for anyone using it after doing a `cargo install lalrpop` to know which version they have and if they should update by having a `--version` and/or `-V` option like most commands have.

**Comment by nikomatsakis** [comment]
Agreed. Marking as "good first bug". To fix, you would want to edit `lalrpop/src/main.rs` and modify [the `USAGE` string](https://github.com/nikomatsakis/lalrpop/blob/master/lalrpop/src/main.rs#L67-L78) to include a `-v`. You could then modify [the args struct](https://github.com/nikomatsakis/lalrpop/blob/master/lalrpop/src/main.rs#L80-L88) to have a `flag_verbose` field or something like that. The [Docopt](http://docs.rs/docopt) package should then link those up, so you can modify the `main` code to print out the current version. 

One slight annoyance will be keeping it in sync. For this, we probably want to update the [version.sh script](https://github.com/nikomatsakis/lalrpop/blob/master/version.sh) to add another line at the end that will automatically update it, I *suppose*. Is there a better way? I imagine modifying `version.sh` might look something like this:

```bash
perl -p -i -e 's/const VERSION: &'static str = "[0-9.]+";$/const VERSION: &'static str = "'$1'";' \
    lalrpop/src/main.rs
```

**Comment by utkarshkukreti** [comment]
How about using [`env!("CARGO_PKG_VERSION")`](http://doc.crates.io/environment-variables.html#environment-variables-cargo-sets-for-crates)?

**Comment by Dragonrun1** [comment]
Cargo macro would make more sense to me then it can't really be forgotten ;)

**Comment by nikomatsakis** [comment]
Yeah, `env!` seems better.

**Comment by jdanford** [comment]
This can be closed thanks to #235, right?

**Comment by nikomatsakis** [comment]
Yep.
